### PR TITLE
IA Redesign Blogpost

### PR DESCRIPTION
**Pull request by shannonbux**

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview for *using-postcss-sass* failed.

Built with commit efea0cfa4a5bc589c9a6e6178539dc0ef8394243

https://app.netlify.com/sites/using-postcss-sass/deploys/5b5fbe044ed62f552a3c1732

**Comment by KyleAMathews** [comment]
Deploy preview for *image-processing* failed.

Built with commit efea0cfa4a5bc589c9a6e6178539dc0ef8394243

https://app.netlify.com/sites/image-processing/deploys/5b5fbe054ed62f552a3c1736

**Comment by KyleAMathews** [comment]
Deploy preview for *using-contentful* failed.

Built with commit efea0cfa4a5bc589c9a6e6178539dc0ef8394243

https://app.netlify.com/sites/using-contentful/deploys/5b5fbe064ed62f552a3c173c

File: docs/blog/2018-07-31-docs-redesign/index.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+Solution:
+
+- Add a recipes section to both the Getting Started and READme file with a list of how to do basic tasks very quickly
+
+Common obstacle #4
+
+- There is an invisible relationship between categories of docs like this:
+  Core Concepts > Guides > Tutorials
+  Getting Started > Recipes > Guides > Tutorials
```
[the remark this answers is not in the document]
Comment (shannonbux) [reply docs/blog/2018-07-31-docs-redesign/index.md]: haha oh thanks for the questions! 

The invisible relationship is the connections between information (i.e. Core Concepts lays a foundation of principles of how things work, and then guides provide specific examples of how those principles work in action, and then tutorials provide the most real-life embodiment of the core concepts. I guess it's like building a house. Foundation first, then walls and structure, then finishing it so people can actually walk through it and live in it.

This informational structure also reflects one ideal pathway through the docs (probably there's more than one)

I'll update this! Thanks!

[the remark this answers is not in the document]
Comment (shannonbux) [reply docs/blog/2018-07-31-docs-redesign/index.md]: Although...maybe I just simplify things. I could be getting WAY too meta here lol

File: docs/blog/2018-07-31-docs-redesign/index.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+Possible solutions:
+
+- use Google data over Algolia search data
+- work on the UI for Algolia search so it pulls up what people need
+- dedicated Algolia search page
+
+Common obstacle #2:
+
+- Information embedded in the tutorial doesn't _also_ exist in the guides, like "how to create links between pages." Algolia search doesn't find keyword inside headers within a document and Google search didn't surface this as well as guides. [TRUE???]
```
Comment (KyleAMathews) [inline docs/blog/2018-07-31-docs-redesign/index.md]: not really sure about this — perhaps @Haroenv can help?

**Comment by KyleAMathews** [comment]
Deploy preview for *using-styled-components* failed.

Built with commit e1aa4909bd54895c6311a247f82272c58294eacf

https://app.netlify.com/sites/using-styled-components/deploys/5b60b7eb1f12b773526546d0

File: docs/blog/2018-07-31-docs-redesign/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (m-allanson) [reply docs/blog/2018-07-31-docs-redesign/index.md]: That make sense, thanks! Your updated version is a good condensed summary without getting into the details too.

File: docs/blog/2018-07-31-docs-redesign/index.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+One thing that happened (and this shouldn't surprise me) is that Gatsby community members are enthusiastic and willing to help. I included an invitation at the end of the card sort that invited card sort participants to sign up for a usability test, thinking "oh, no one will sign up for this." Then, about 15 people signed up, and I only need about 5-7 for solid data ([reference here](https://www.invisionapp.com/blog/ux-usability-research-testing/)).
+
+So, I did three different usability tests with about 5 people each and got some really valuable insights. Thank you to all these people for taking the time to meet with me for usability tests!
+
+Thank you Martyn Hoyer, Eka, Benjamin Modayil, Nicky Meuleman, Jonathan Prozzi, Kelly Lawrence, Chris Wiseman, Simon Koelewijn, Cameron Steele, Bogdan Lazar, Shannon Smith, Hugo Marques, Ria Carmin, Abhishek Vishwakarma, Peter Wiebe, and Korey Boone.
+
+Common obstacle #1:
+
+- Information embedded in the tutorial doesn't _also_ exist in the guides, like "how to create links between pages." Algolia search doesn't find keyword inside headers within a document.
```
Comment (Haroenv) [inline docs/blog/2018-07-31-docs-redesign/index.md]: cc @s-pace, this is probably something misconfigured in the docsearch

Comment (s-pace) [reply docs/blog/2018-07-31-docs-redesign/index.md]: This is not true indeed.

We do match element from lvlX elements!

Comment (shannonbux) [reply docs/blog/2018-07-31-docs-redesign/index.md]: ok thanks I'll edit this! You're right; I retested it and released that it was just a readability problem. I tried searching for "slugs" and the tutorial header showed up lower on the list and was so wordy, it was hard to scan. So yeah, there is no functionality problem, only a problem with how long the text is and it's basically too long for such a small box, so very hard to read quickly:

screen shot 2018-08-01 at 10 19 00 am

**Comment by shannonbux** [comment]
cherry-picked to v1

**Comment by s-pace** [comment]
👏

**Pull request merged**